### PR TITLE
Kms 19891 submission flow

### PR DESCRIPTION
**Pull request by Channa53**

### CODE REVIEW

File: local/kaltura/lang/en/local_kaltura.php [hunk 1]
```diff
@@ -70,4 +70,6 @@
 $string['time'] = 'Time';
 $string['trace_log'] = 'Enable trace logging';
 $string['trace_log_desc'] = 'If enabled, all requests and responses to and from Kaltura are logged.  These logs can be used by Kaltura support to diagnose any problems encountered.  Enabling this setting may affect Moodle performance.  You may download a CSV file of the logs from <a href="{$a}">here</a>.';
-$string['browse_and_embed'] = 'Browse and Embed';
\ No newline at end of file
+$string['browse_and_embed'] = 'Browse and Embed';
+$string['enable_assignment'] = 'Enable Assignment';
+$string['enable_assignment_desc'] = 'Enable mashup tool for assignment video';
```
Comment (muli) [inline local/kaltura/lang/en/local_kaltura.php]: "Mashup" tool is not an official term in Moodle.
Also, I believe that the description here does not really help understand what this setting means.

Comment (Channa53) [reply local/kaltura/lang/en/local_kaltura.php]: yeah, it is temporarily, I'm waiting for Ido to decide.. thanks

**Comment by Channa53** [comment]
@omridevk ping

**Comment by omridevk** [comment]
LGTM, @muli?

**Review by muli — APPROVED** [review]

**Pull request merged**